compiler/src/test/scala/org/scalajs/core/compiler/test/JSExportTest.scala

```diff
@@ -433,127 +471,75 @@ class JSExportTest extends DirectTest with TestHelpers {
   }
 
   @Test
-  def noImplicitNameNestedExportClass: Unit = {
```

Comment:
:P Indeed, this actually doesn't work because of the special casing of `@JSExport` in nested things... Maybe we should just drop it :S